Assets/Scripts/UI/TwitchLeaderboard.cs

```diff
@@ -31,50 +27,74 @@ private void Start()
             return;
         }
 
-        float delay = 0.6f;
-        int index = 0;
+        if (leaderboard.Success)
+        {
+            promptTransform.FindChild("Retry").gameObject.SetActive(false);
```

Comment:
This looks nasty. Could you not just have a reference direct to that Retry GameObject (rather than Transform) assigned in the inspector, and then it's just a nice simple `RetryText.SetActive(false)`? In fact, all of the uses of `promptTransform` I see are in fact GameObject uses instead.